### DISCUSSION TITLE
Options that are not in the in-game option list.

### DISCUSSION THREAD

**Discussion by Noomene** [angband/angband#5632] [General]
Sorry for having to ask here. I couldn't register on the forums. I asked on reddit and I got basically 0 useful answers. So I'm asking here and I hope it's not too bothering.

There is an option in the manual called [always_repeat](https://angband.readthedocs.io/en/latest/playing.html#command-counts). The manual says that it can be changed. I looked in the prefs files, and in the in-game option list but couldn't find anywhere where that option can be set or reset. I would be very grateful if you can give me some hint on how I can change this option.

Sorry again for having to ask here.

**Comment by NickMcConnell** [comment]
The manual is in fact wrong - auto-repeat is set (to 99) for the relevant commands now.
What happened when you tried to register for the forums?

**Reply by Noomene** [reply]
Thanks for your reply. When you try to register, there is a random question on the registration screen that is, I think, unanswerable. The random question is: "New accounts are closed, sorry" ... I don't know if there is an actual answer to that and I'm missing something, or if it is just a message from the admin.

So thanks for your answer again, if the manual is not up to date then it makes sense that I couldn't find that option. 
Just to be absolutely sure, if I understood you correctly, 99 is hard-coded and cannot be changed through the usual options. Would touching the code and recompiling it work? If so, where should I look?

Thank you so much again for your time and your reply.

**Reply by NickMcConnell** [reply]
The relevant code is in cmd-core.c - a search for 99 in that file will find it quickly.  Certainly changing that and recompiling would work.

I'll look into the forum issue.

**Reply by Noomene** [reply]
Thank you sooo much for your kind reply and for your time. This is going to be the first time I ever touch some game code ever. Will be fun.
Have a wonderful week.
